files/magneticmediajs/info.ini

```diff
@@ -2,4 +2,4 @@ author = "Arnaud Leyder"
 github = "https://github.com/arnaudleyder/magneticmediajs"
 homepage = "https://www.magneticmediajs.com/"
 description = "JavaScript And CSS To Nicely Display Media Content"
-mainfile = "mjs.min.js"
\ No newline at end of file
+mainfile = "js/mjs.min.js"
```

Comment:
Sounds good, can you update the existing directories with `mainfile` files. See https://github.com/jsdelivr/jsdelivr/blob/master/files/jsface/0.1/mainfile for reference.

This will prevent existing urls from breaking for your project